packages/codebyte-editor/.storybook/main.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Discussing in our pairing: we should have a standalone storybook package so there's only one, instead of stories per package.